h2o-algos/src/main/java/hex/glm/GLMModel.java

```diff
@@ -28,6 +28,8 @@
  */
 public class GLMModel extends Model<GLMModel,GLMModel.GLMParameters,GLMModel.GLMOutput> {
 
+  final static public double _EPS = 1e-6;
+  final static public double _OneOEPS = 1e6;
   public GLMModel(Key selfKey, GLMParameters parms, GLM job, double [] ymu, double ySigma, double lambda_max, long nobs) {
```

Comment:
I am sick and tired of setting a value to 1e-6 if it is 0.  Hence, I use a constant EPS to replace the 1e-6.  Doing it this way, we can change it to bigger or smaller at one place.